lib/error-handlers/string.js

```diff
@@ -30,13 +30,17 @@ var isString = require('lodash').isString;
  */
 function handleStringAsError(err, errorMessage) {
   var fauxError = new Error();
-  var errChecked = fauxError.stack;
+  var fullStack = fauxError.stack.split('\n');
+  var cleanedStack = fullStack.slice(0, 1).concat(fullStack.slice(4));
```

Comment:
Slicing out particular numbers of stack frames could become incorrect as new functions are added. Could we add a test checking that the correct number of stack frames are omitted?